hw2/04-background.js

```diff
@@ -0,0 +1,27 @@
+let timeInterval = setInterval(function () {
+  let randomColor1 = Math.floor(Math.random() * 16777215).toString(10);
```

Comment:
It's a nitpick, but you could use a template literal here instead of `toString()`. I don't know how that breaks down in terms of other nitpicks like efficiency, but it would probably be slightly more readable.